frontend/benefit/applicant/src/components/mainIngress/MainIngress.tsx

```diff
@@ -24,36 +25,38 @@ const MainIngress: React.FC = () => {
   ));
 
   return (
-    <Container
-      backgroundColor={theme.colors.silverLight}
-      data-testid="main-ingress"
-    >
+    <div data-testid="main-ingress">
       <$Container>
-        <$Heading>{t('common:mainIngress.heading')}</$Heading>
-        {notificationItems}
-        <$TextContainer>
-          <$Description>
-            {t('common:mainIngress.description1')}
-            {/* TODO: uncomment once having link to redirect to more info url,
+        <Container>
+          <$Heading>{t('common:mainIngress.heading')}</$Heading>
+          {notificationItems}
+          <$TextContainer>
+            <$Description>
+              {t('common:mainIngress.description1')}
+              {/* TODO: uncomment once having link to redirect to more info url,
             handleMoreInfoClick is from useMainIngress */}
-            {/* <$Link onClick={handleMoreInfoClick}>
+              {/* <$Link onClick={handleMoreInfoClick}>
```

Comment:
This is ok for me and I see the point in constant structural improvement.

In general, we've previously discussed with @rikuke and @mjturt that commented code should be left out. An example of this is that there's plenty of badly aged TODO's in the code (especially in Django side) that the current team has no idea of.